src/quantities.js

```diff
@@ -235,8 +235,12 @@ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
     "<rpm>"   :[["rpm"], 2.0*Math.PI / 60.0, "angular_velocity", ["<radian>"], ["<second>"]],
 
     /* memory */
-    "<byte>"  :[["B","byte"], 1.0, "memory", ["<byte>"]],
-    "<bit>"  :[["b","bit"], 0.125, "memory", ["<byte>"]],
+    "<byte>"  :[["B","byte","bytes"], 1.0, "memory", ["<byte>"]],
+    "<bit>"  :[["b","bit","bits"], 0.125, "memory", ["<byte>"]],
+
+    /* memory speed */
+    "<Bps>" : [["Bps"], 1.0, "speed", ["<byte>"], ["<second>"]],
```

Comment:
`Bps` is not a kind of `speed` (It cannot be converted to `m/h` for instance). `speed` is length over time.